### PR TITLE
add relativeValue and monetaryValue.csv

### LINKED ISSUE #81: participationFees: Proportional fees
In Kazakhstan participation fees have a proportional expression, presumably of either the tender or award value, depending on the type of fee, though I wasn't able to find an example due to technical issues with the [Kazakh e-procurement portal](https://goszakup.gov.kz/).

For fees calculated on the tender value, the fee value could be calculated and provided in `participationFee.value`, however for fees calculated on the award value this wouldn't be possible until after the fact, so a new field on `participationFee` may be required to express the fee percentage.

### PR DESCRIPTION
**Pull request by odscjen**
closes https://github.com/open-contracting/ocds-extensions/issues/81

### CODE REVIEW

File: release-schema.json [hunk 1]
```diff
@@ -73,6 +78,38 @@
...
+      ],
+      "properties": {
+        "percentage": {
+          "title": "Percentage",
+          "description": "The percentage of the monetary value.",
+          "type": [
+            "number"
+          ],
+          "maximum": 1,
+          "minimum": 0,
```
Comment (duncandewhurst) [inline release-schema.json:98]: Noting that this differs from the suggestion in https://github.com/open-contracting/ocds-extensions/issues/81#issuecomment-1830975752 (0 to 100) but is in keeping with the modelling of percentages in the subcontracting extension so I think it's all good.

File: README.md [hunk 2]
```diff
@@ -52,6 +52,32 @@ The following JSON snippet models a contracting process where fees are applied t
 }
 ```
 
+The following JSON snippet models a contracting process where fees are calculated as a percentage of the awarded contract value.
```
Comment (duncandewhurst) [inline README.md]: Illustrate how percentages are modelled and explain the meaning of 'win'

```suggestion
The following JSON snippet models a contracting process with a participation fee of 5% of the award value, payable by the winning bidder.
```

**Review by duncandewhurst — CHANGES_REQUESTED** [review]
Happy for you to request a review from James once my suggestion is resolved.

File: release-schema.json (hunk 1 above)
Comment (odscjen) [reply release-schema.json:98]: it's also how `bid.statistic.value` says to provide percentages ("Percentages should be provided as a proportion of 1 (e.g. 10% = 0.1)"), and `operatorRevenueShare` from Contract Terms and `shareholders.shareholding` from Shareholders (both of which are percentages) are modelled.

Comment (jpmckinney) [reply release-schema.json:98]: Hmm, we probably need to edit many of these to add ", expressed as a decimal proportion between 0 and 1."

Can you make that change here and open an issue to update [contractTerms](https://github.com/open-contracting-extensions/ocds_contractTerms_extension/pull/7), [shareholders](https://github.com/open-contracting-extensions/ocds_shareholders_extension/pull/27), ✅︎subcontracting, ✅︎bids, and [ppp](https://github.com/open-contracting/ocds-extensions/issues/214)?

In this extension, we can also rename `percentage` to `proportion`.

In contractTerms, the description of `operatorRevenueShare` should also replace "percentage" with "proportion". Same for `shareholding`, in shareholders.

✅︎ bids can be updated to:

> A total count should be expressed as an integer. A proportion should be expressed as a decimal (for example, 0.1) and not as a percentage (for example, 10).

and 🕗ppp to:

> , expressed as a decimal (for example, 0.1) and not as a percentage (for example, 10).

Can also delete the "further details" sentence while we're editing ppp.

File: README.md [hunk 3]
```diff
@@ -52,6 +52,32 @@ The following JSON snippet models a contracting process where fees are applied t
 }
 ```
 
+The following JSON snippet models a contracting process with a participation fee of 5% of the award value, payable by the winning bidder.
```
Comment (jpmckinney) [inline README.md]: Make similar changes to the existing text for the previous example:

```suggestion
A participation fee of 5% of the award value, payable by the winning bidder:
```

File: release-schema.json [hunk 4]
```diff
@@ -37,6 +37,11 @@
           "description": "The monetary value of this fee.",
           "$ref": "#/definitions/Value"
         },
+        "relativeValue": {
+          "title": "Relative value",
+          "description": "The value of the participation fee, as a proportion of the monetary value.",
```
Comment (jpmckinney) [inline release-schema.json]:
```suggestion
          "description": "The value of the participation fee, as a proportion of a monetary value.",
```

File: release-schema.json [hunk 5]
```diff
@@ -73,6 +78,38 @@
...
+      "description": "The value of the participation fee, as a proportion of a monetary value.",
+      "type": "object",
+      "required": [
+        "proportion",
+        "monetaryValue"
+      ],
+      "properties": {
+        "proportion": {
+          "title": "Proportion",
+          "description": "The percentage of the monetary value, expressed as a decimal proportion between 0 and 1.",
```
Comment (jpmckinney) [inline release-schema.json]:
```suggestion
          "description": "The proportion of the monetary value, expressed as a decimal between 0 and 1.",
```

**Pull request merged**